ui/job-view/details/tabs/PerformanceTab.jsx

```diff
@@ -296,7 +208,12 @@ class PerformanceTab extends React.PureComponent {
             </Alert>
           ) : null
         }
-        {this.maybeRenderPerfData()}
+        {perfJobDetail && (
```

Comment:
Minor change, keep the same logic as before e.g. `perfJobDetail.length === 0` to avoid unexpected behavior.